notebooks/01a-cosmos_tutorial.ipynb

```diff
@@ -213,9 +215,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
+   "display_name": "btk",
```

Comment:
Be careful, this should not be changed in the git repo.

Reply:
this seems to be coming from my jupyter kernel... 
Is this also supposed to be fixed by pre-commit?

Reply:
No. You should take care to change it back to Python 3 before committing ; I don't know of a way to go around this.

Reply:
Ok, I can do it this way then!
I am having a bit of trouble understanding why exactly this is required though... The metadata of the notebook won't change anything right? 
My actual concern is that other than this line, there are a few more that have been changed in the metadata. Do you mean that the entire metadata section should remain the same? or just this line?

Reply:
Maybe the `databooks` pre-commit hook is the best tool for our needs 
https://github.com/datarootsio/databooks
available in https://pre-commit.com/hooks.html

But it will strip the metadata, meaning that the user will have to choose a kernel when the server is starting.
(that's not necessarily bad)

Reply:
Let me know what you think @thuiop and @ismael-mendoza

Reply:
This does seem like something which would be useful.

Reply:
Good point, don't know that. Needs to be checked

Reply:
Please @b-biswas write an issue about that so it can be checked in a future work.